### PR TITLE
Prepare release: 0.99.0

### PR DESCRIPTION
**Pull request by alejoe91**
- added release notes
- commented out "git" installs
- bumped up version

### CODE REVIEW

File: doc/releases/0.99.0.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+SpikeInterface 0.99.0 release notes
+-----------------------------------
+
+31th October 2023
+
+Main changes:
+
+* Refactored :code:`core.generate` module to create high-quality lazy ground-truth data
+* Added DeepInterpolation training, transfer, and inference to preprocessing
+* Added new quality_metrics (syncrhrony, firing_range, amplitude_cv)
```
Comment (zm711) [inline doc/releases/0.99.0.rst:12]:
```suggestion
* Added new quality_metrics (synchrony, firing_range, amplitude_cv)
```

File: doc/whatisnew.rst [hunk 2]
```diff
@@ -31,6 +32,18 @@ Release notes
   releases/0.9.1.rst
 
 
+Version 0.99.0
+==============
+
+* Refactored :code:`core.generate` module to create high-quality lazy ground-truth data
+* Added DeepInterpolation training, transfer, and inference to preprocessing
+* Added new quality_metrics (syncrhrony, firing_range, amplitude_cv)
```
Comment (zm711) [inline doc/whatisnew.rst:40]:
```suggestion
* Added new quality_metrics (synchrony, firing_range, amplitude_cv)
```

**Comment by h-mayorquin** [comment]
Neuroconv tests:
https://github.com/catalystneuro/neuroconv/pull/627

File: doc/releases/0.99.0.rst [hunk 3]
```diff
@@ -0,0 +1,143 @@
+.. _release0.99.0:
+
+SpikeInterface 0.99.0 release notes
+-----------------------------------
+
+31th October 2023
```
Comment (alejoe91) [inline doc/releases/0.99.0.rst:6]:
```suggestion
3rd November 2023
```

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**